test/models/audit_trail_test.rb

```diff
@@ -1,66 +1,66 @@
-require 'test_helper'
+# require 'test_helper'
```

Comment:
mongo-related, so I feel comfortable just commenting out until we remove these files.